Editing Readme.md file

Lahirutech opened the pull request:
Fixes #

## Proposed Changes

  -
  -
  -

agentmilindu commented on lib/core/config.js:
```diff
@@ -18,7 +17,7 @@ class Config {
    */
   readConfig(provider) {
     if (provider === providers.AWS) {
-      return this.parseAWS();
```
I hope you are very sure the `return` here is implicit and this change would not affect the functionality.

dmbutalsodm commented:
Fixed, I misunderstood that section of the code, apologies.

Lahirutech committed:
Adding Kubernetes Service in reademe.md

Adding Kubernetes Service in reademe.md forAmazon Azure and GCP

Lahirutech committed:
Adding Azure Files  in readme.md

Found this service in this documentation
https://docs.microsoft.com/en-us/azure/storage/common/storage-introduction#azure-files

Lahirutech committed:
Adding Load Balancing service in readme.md

Found in here
https://cloud.google.com/load-balancing/docs/

Lahirutech committed:
adding peering for GCP in readme.md

Adding Direct Peering and Carrier peering in GCP
Found in here
https://cloud.google.com/hybrid-connectivity/

Lahirutech committed:
Adding * mark feature in readme.md

* marked features are to be implemented

Lahirutech committed:
Adding Azure DNS service to readme.md

Azure DNS documentation found in here
https://azure.microsoft.com/en-us/services/dns/

Lahirutech committed:
Adding cloud SQL to readme.md

Cloud SQL service is added after getting this documentation
https://cloud.google.com/sql/docs/
